cypress/e2e/SauceDemo/UI/Tests/LandingPageTests.cy.ts

```diff
@@ -11,7 +12,7 @@ describe("Landing Page Tests", () => {
       settings.sauce_demo.credentials.password
     );
   });
-  it("Verify default sorting on landing page", () => {
+  it("Verify default sorng on landing page", () => {
```

Comment:
it seems we have little typo here on sorng

Reply:
fixed. Good catch!